### ISSUE TITLE
Add protractor tests to linter

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/cfgov-refresh#723]
Protractor tests currently aren't linted. `'<%= loc.test %>/browser_tests/**/*.js'` should be added to https://github.com/cfpb/cfgov-refresh/blob/flapjack/Gruntfile.js#L283

There are currently 80+ linter offenses in the JS. If any of these make sense to keep in the tests, add a linter override file in (like [this](https://github.com/cfpb/cfgov-refresh/blob/flapjack/_tests/unit_tests/.eslintrc)) in `_tests/browser_tests`.

**Comment by KimberlyMunoz** [comment]
Some of the offenses are from Jasmine's syntax, so we're going to need an .eslintrc file.

One thing that I think might be controversial is the a lot of our "warnings" are from max-length. Should we ignore this rule on tests since tests are a little different.

I'm thinking of two scenarios that would be awkward to add line breaks.
~~~
var StudentsAndGraduates = require( '../../page_objects/page_careers-students-and-graduates.js' );
~~~
or
~~~
it( 'should include 3 bureau missions, Educate, Enforce, Empower', function() {
      expect( page.missions.getText() ).toEqual['Educate', 'Enforce', 'Empower'];
  } );
~~~

**Comment by KimberlyMunoz** [comment]
Closed by #745
